debugger-launchjson.md

```diff
@@ -45,7 +45,7 @@ The default launch.json template (as of C# extension v1.20.0) for ASP.NET Core p
 ```json
     "serverReadyAction": {
         "action": "openExternally",
-        "pattern": "^\\s*Now listening on:\\s+(https?://\\S+)"
+        "pattern": "\\s*Now listening on:\\s+(https?://\\S+)"
```

Comment:
Assuming JavaScript supports it, this should probably be `\\bNow...` to make sure there is a word boundary. Can you test if that works in your scenario?

Reply:
Sure I'll test it today. I didn't know the default template output this. I still had the launch browser property. I can update the template if you point me towards it.